src/components/pages/Evaluation/DefineScopePage.svelte

```diff
@@ -156,9 +159,44 @@
       return available.indexOf(assertion.test.num) == -1;
     });
 
-    assertionsToRemove.forEach((assertion) => {
-      assertions.remove(assertion);
-    });
+    if(assertionsToRemove.length > 0){
+      let answeredCount = 0;
+      assertionsToRemove.forEach((assertion) => {
+        if(assertion.result.outcome.id != "earl:untested"){
+          answeredCount++;
+        }
+      });
+
+      if(answeredCount > 0){
+        if (confirm(TRANSLATED.DATA_LOSS_WARNING)) {
+          assertionsToRemove.forEach((assertion) => {
+            assertions.remove(assertion);
+          });
+        } else {
+          $scopeStore['WCAG_VERSION'] = oldwcag;
+          $scopeStore['CONFORMANCE_TARGET'] = oldtarget;
+        }
+      }else{
+        assertionsToRemove.forEach((assertion) => {
+          assertions.remove(assertion);
+        });
+      }
+    }
+    
   }  
 
+  let oldwcag = "";
+  let oldtarget = "";
+  onMount(() => {
+    oldwcag = document.getElementById("wcag_version").value;
```

Comment:
We should be able to get this from the store that this value syncs with?

And we can get  a reference to element using `bind:this` in the template that contains `wcag_version`?